Fix running smoketests against a remote server requiring spacetime auth (#5455)
clockworklabs/SpacetimeDB: tools/ci/README.md

When specified, tests will connect to the given URL instead of starting local server instances. Tests that require local server control (like restart tests) will be skipped.

- `--spacetime-login <SPACETIME_LOGIN>`: Use a SpacetimeAuth-issued login for remote-server tests.

> This is required for maincloud and maincloud staging, which reject direct server-issued logins for privileged operations.

Reviewer: ```suggestion

This is required for servers that reject direct server-issued logins for privileged operations.
```

> Optionally accepts an auth host to pass through to `spacetime login`, for example `--spacetime-login=https://maincloud.staging.spacetimedb.com`.

Reviewer: ```suggestion
Optionally accepts an auth host to pass through to `spacetime login`, for example `--spacetime-login=https://spacetimedb.com`.
```
